src/server/services/avg/types.ts

```diff
@@ -11,14 +11,15 @@ export type AVGRequest = {
   status: string;
   registratieDatum: string;
   type: string;
-  thema: string;
   onderwerp: string;
+  toelichting: string;
   resultaat: string;
   ontvangstDatum: string;
   opschortenGestartOp: string;
   datumInBehandeling: string;
   datumAfhandeling: string;
   link: LinkProps;
+  themas?: (string | null)[];

```

Comment:
Misschien kunnen we dit iets sterker typen.

```suggestion
  themas: string[] | null;
```

`null` zou je ook weg kunnen laten en in case of error van themas response een lege array teruggeven. Is misschien relaxter programmeren in de FE. 

Als je wel `null` wilt kun je checken op:

```
themas === null ? 'Thema's onbekend' : themas,
```